cmd/nerdctl/save_linux_test.go

```diff
@@ -48,6 +48,43 @@ func TestSave(t *testing.T) {
 	assert.Assert(t, strings.Contains(etcOSRelease, "Alpine"))
 }
 
+func TestSaveById(t *testing.T) {
+	base := testutil.NewBase(t)
+	base.Cmd("pull", testutil.AlpineImage).AssertOK()
+	inspect := base.InspectImage(testutil.AlpineImage)
```

Comment:
```suggestion
	inspect := base.InspectImage(testutil.Commonmage)
```

And the test can be moved to `save_test.go`

Reply:
fixed